### DISCUSSION TITLE
How to display nested cards in the Admin UI?

### DISCUSSION THREAD

**Discussion by posthardcode** [keystonejs/keystone#7534] [Questions] [answered]
Hey team 👋 

In my `User` list, I have a `companyInfo` field with a relationship to `Company.owner`, and in my `Company` list, I have an `owner` field with a relationship to `User.companyInfo`. `Company` has an `address` field with a relationship to `Address`.

When editing `Company info`, under the User list view, I can see the `address` being displayed as a card view:

<img width="737" alt="Screenshot 2022-05-15 at 14 40 17" src="https://user-images.githubusercontent.com/8139686/168473598-0532d206-342f-406b-9f74-f7b08702d55b.png">

However, if not editing, `address` is displayed as a link:

<img width="750" alt="Screenshot 2022-05-15 at 14 40 03" src="https://user-images.githubusercontent.com/8139686/168473630-f93dc707-0f97-4a2d-b8ea-14b82a42d9c7.png">

Is there a way to always display `address` as a card view outside of editing (or, in general, how can we display "nested cards")?

**Comment by gwyneplaine** [comment, marked as answer]
@posthardcode thanks for raising this with us, we'll be looking at improving the relationship card UI soon, and this behaviour is something we'll be improving.
